macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -96,10 +93,10 @@
         {%- if column_monitors %}
             {% for monitor in column_monitors %}
                 select
-                    edr_column_name,
+                    {{ elementary.const_as_string(column_obj.name) }} as edr_column_name,
                     bucket_start,
                     bucket_end,
-                    bucket_duration_hours,
+                    {{ elementary.timediff("hour", "bucket_start", "bucket_end") }} as bucket_duration_hours,
```

Comment:
Is this correct for the no timestamp case?
(because previously we differentiated between the two and set the value to null in this case)

Reply:
Fixed [here](https://github.com/elementary-data/dbt-data-reliability/pull/728/commits/19a287df4eb48465f9d6255b607367364b0708a3).